content/en/blog/releases/v0.6.0.md

```diff
@@ -0,0 +1,78 @@
+
+The feature list is impressive. It made it hard to pick some top 3. Here is my choice:
+
+## Parameterization of Builds
+
+Our API objects; the build strategy that defines the steps to turn source code into a container image, the build where a user configures his container image build, and the build run as an instance of a build; so far suffered of a major capability: parameterization. We defined the available inputs that a user specified in a build. Those were only the minimum: the location of the source code, the name of the strategy and the destination image. Any other customization of the build was not possible, limiting the possible scenarios.
+
+Two major enhancements bring us a huge step forward:
+
+Build strategy authors can now define parameters that they can use then in their steps. Build users provide the values for those parameters. Our sample build strategies start to make use of this feature: the [BuildKit sample](https://github.com/shipwright-io/build/blob/v0.6.0/samples/buildstrategy/buildkit/buildstrategy_buildkit_cr.yaml#L14-L16) now contains a parameter to enable the usage of insecure container registry, and the [ko sample](https://github.com/shipwright-io/build/blob/v0.6.0/samples/buildstrategy/ko/buildstrategy_ko_cr.yaml#L8-L22) comes with a rich set of parameters to select Go version and flags, ko version and the main package directory. The [ko sample build](https://github.com/shipwright-io/build/blob/v0.6.0/samples/build/build_ko_cr.yaml#L10-L15) outlines how those parameters can be used to build the Shipwright Build controller image. Read more about parameters in the documentation for [build strategy authors](https://github.com/shipwright-io/build/blob/v0.6.0/docs/buildstrategies.md#strategy-parameters) and [build users](https://github.com/shipwright-io/build/blob/v0.6.0/docs/build.md#defining-paramvalues).
+
+But, not in all scenarios, it is possible to define all possible parameters upfront in a build strategy. For those tools that control behavior through environment variables, users can now define environment variables in their builds and build runs. Environment variable values can be referenced from Kubernetes secrets to support scenarios where for example tokens for private dependency management systems are needed. Examples of builds and build runs with environment variables can be found in our [build](https://github.com/shipwright-io/build/blob/v0.6.0/docs/build.md) and [build run](https://github.com/shipwright-io/build/blob/v0.6.0/docs/buildrun.md) documentation.
+
+## CLI
+
+Being based on Kubernetes, all our objects are Kubernetes custom resources with their ecosystem-specific syntax and concepts. But, not all our users should require a black belt in using Kubernetes. We are therefore proud to release the first version of our CLI. Build users can use it to easily manage their builds and build runs. Some highlights:
```

Comment:
```suggestion
Being based on Kubernetes, all our objects are Kubernetes custom resources with their ecosystem-specific syntax and concepts. But, our users should not be required to be experts in using Kubernetes. We are therefore proud to release the first version of our CLI. Build users can use it to easily manage their builds and build runs. Some highlights:
```

Reply:
It is only a suggestion, I usually shy away from metaphors and local colloquialisms as not everyone understands their meaning.